src/test/java/com/googleinterns/zoomtube/servlets/LectureServletTest.java

```diff
@@ -67,6 +67,7 @@
   private PrintWriter writer;
 
   private static final String LINK_INPUT = "link-input";
```

Comment:
should link-input be camel case?

Reply:
link-input is a the input name in the HTML

The style guide didn't include conventions for naming inputs so I defaulted to the convention for ID and class naming.
https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters